cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -615,6 +621,10 @@ spec:
               mountPath: /var/configs/etl/federated
               readOnly: true
             {{- end }}
+            {{- if .Values.kubecostModel.caCertsSecret }}
+            - name: ca-certs-secret
+              mountPath: /etc/pki/ca-trust/source/anchors
```

Comment:
This is a good start, but are we sure this should be hard-coded? This mountPath should probably be a helm config? I think that different OS's will look at different paths for its trust store. For example `/etc/ssl/certs`, or `/etc/pki/tls/certs`, or `/usr/local/share/ca-certificates/`?

Reply:
Based on the information from the [Fedora documentation](https://docs.fedoraproject.org/en-US/quick-docs/using-shared-system-certificates/#_adding_new_certificates) and the user's request, for now, we only need to mount the certificate to `/etc/pki/ca-trust/source/anchors`. While other scenarios might require different mount paths in the future, this setup works well for our current needs. We can include it in the Helm config as the default for now but it might remain unused. thoughts?

Reply:
Ok, let's leave it hard-coded for now due to user-specific request. Let's make a task to update it in the future?

Reply:
sure :+1: